misk/src/test/kotlin/misk/web/actions/WebActionMetadataActionTest.kt

```diff
@@ -27,6 +29,12 @@ class WebActionMetadataActionTest {
     assertThat(customServiceActionMetadata.pathPattern).isEqualTo("/custom_service_access")
     assertThat(customServiceActionMetadata.allowedServices).containsOnly("payments")
     assertThat(customServiceActionMetadata.allowedRoles).isEmpty()
+    assertThat(
+        customServiceActionMetadata.applicationInterceptors.all {
+          classNameFormatter.isValid(it)
```

Comment:
seems a bit backwards to assert this here; how about instead we confirm that we return the formatted class name?